CHANGES.md

```diff
@@ -1,3 +1,32 @@
+## edge-19.8.6
+
+* CLI
+  * Fixed missing `clusterDomain` in render RenderTapOutputProfile
+  * Added LINKERD2_PROXY_DESTINATION_GET_SUFFIXES env variable (thanks @arminbuerkle!)
+  * Merged the CLI 'installValues' type with Helm 'Values' type 
```

Comment:
All of the indented bullets below this line should be deleted as those are from individual commits within the PR mentioned on line 6. We generally want to summarize the entire PR and describe why it is useful, in a single bullet point. 

And a high-level note, I am not sure if the specific PR listed on line 6 is user-facing enough to include - I would defer to @alpeb

Reply:
That's correct, that's a 100% implementation detail that doesn't change the UX, so it shouldn't go in the release notes.